sample-plugin/pom.xml

```
@@ -66,6 +66,10 @@
             <artifactId>test-harness</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>apache-httpcomponents-client-5-api</artifactId>
```

I believe that this should be in the `test` scope as the dependencies that precede it are.  However, I need advice from those who are more experienced with the sample-plugin pom file, since `apache-httpcomponents-client-4-api` is only included in the `bom-weekly` pom and is not included in the sample plugin.

@jglick can you offer guidance on whether this new component should be in the `test` scope in the sample plugin or if it should only be in the `bom-weekly` pom like `apache-httpcomponents-client-4-api` ?
```suggestion
            <artifactId>apache-httpcomponents-client-5-api</artifactId>
            <scope>test</scope>
```

---

It can be added to `test` scope. However better still it is not add it at all and have it be picked up as a transitive dependency of some feature plugin that uses it and which is also in the BOM. Is there one?

---

I'm currently in the process of creating PR to plugins that may pick it up. Currently the only usage is in the upcoming release of [jenkinsci/custom-folder-icon-plugin](https://github.com/jenkinsci/custom-folder-icon-plugin/) where it is a `test` dependency.
So...shall I remove it from the sample-plugin?